.github/workflows/build.yaml

```diff
@@ -0,0 +1,32 @@
+name: Node Build
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js 14
+      uses: actions/setup-node@v1
+      with:
+        node-version: 14
+    - run: npm install
+    - run: npm run build
+      env:
+        GATSBY_SEGMENT_WRITE_KEY: ${{ secrets.GATSBY_SEGMENT_WRITE_KEY }}
+        GATSBY_TRACK_PAGE: true
+        GATSBY_DRIFT: ${{ secrets.GATSBY_DRIFT }}
+    - name: Deploy to Github Page
+      uses: peaceiris/actions-gh-pages@v3
+      with:
+        publish_dir: ./public
+        keep_files: true
+        personal_token: ${{ secrets.DEPLOY_GITHUB_TOKEN }}
+        external_repository: kintohub/www.kintohub.com
```

Comment:
action: this is not necessary, since we are deploying to the same repo
https://github.com/peaceiris/actions-gh-pages#%EF%B8%8F-deploy-to-external-repository-external_repository